docker/scvi/scripts/main/annotate_cells.py

```diff
@@ -8,101 +8,109 @@
 import scvi
 
 
-parser = argparse.ArgumentParser(
-    description='Annotate clusters'
-)
+parser = argparse.ArgumentParser(description="Annotate clusters")
 parser.add_argument(
-    '--adata-input',
-    dest='adata_input',
-    type=str,
-    help='AnnData object for a dataset'
+    "--adata-input", dest="adata_input", type=str, help="AnnData object for a dataset"
 )
 parser.add_argument(
-    '--marker-genes',
-    dest='marker_genes',
+    "--marker-genes",
+    dest="marker_genes",
     type=str,
-    default='resources/celltype_marker_table.csv',
-    help='Path to marker_genes .csv file'
+    default="resources/celltype_marker_table.csv",
```

Comment:
Paths will be removed and the bucket uri will be used in WDL

Reply:
default removed.